tests/test_trade.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from poandy.controller.trade import TradeController
+from poandy.controller.account import AccountController
+
+
+class TestTradeController:
+    def test_get_trades_default(self):
+        account_id = AccountController.get_default_account_id()
+        trades = TradeController.get_trades(account_id)
+        assert isinstance(trades, dict)
+        assert "trades" in trades
+        assert "lastTransactionID" in trades
+        assert isinstance(trades["trades"], list)
+
+    def test_get_trades_with_params(self):
+        account_id = AccountController.get_default_account_id()
+        state = "CLOSED"
+        count = 10
+        instrument = "USD_CAD"
+        trades = TradeController.get_trades(
+            account_id, state=state, count=count, instrument=instrument
+        )
+        if trades["trades"]:
+            assert trades["trades"][0]["state"] == state
+            assert trades["trades"][0]["instrument"] == instrument
+        assert len(trades["trades"]) < count
+
+    def test_get_trades_with_invalid_params(self):
+        account_id = AccountController.get_default_account_id()
+        state = "TEST"
+        count = 600
+        with pytest.raises(ValueError):
+            TradeController.get_trades(account_id, state=state)
+            TradeController.get_trades(account_id, count=count)
+
+    def test_get_open_trades(self):
+        account_id = AccountController.get_default_account_id()
+        open_trades = TradeController.get_open_trades(account_id)
+        assert isinstance(open_trades, dict)
+        assert "trades" in open_trades
+        assert "lastTransactionID" in open_trades
+        assert isinstance(open_trades["trades"], list)
```

Comment:
I think we can also have a test where we test the number of trades we got back (which should be 0, since we didn't do any trades). What do you think @callistachang

Reply:
Not too sure if we should include it in case we make trades in the future. I think it's okay to include for now until the club starts using it?

Reply:
@callistachang @gau-nernst @wowweijie If it's a test account then it should be fine I think. I think the club will be using a different account to trade, they won't be using the same account we use for development.